protoflo/__main__.py

```diff
@@ -15,7 +15,7 @@ def register (user_id, label, ip, port):
 	data = {
 		'type': 'protoflo', 
 		'protocol': 'websocket',
```

Comment:
I guess line 44 needs to be changed too. Is this a change in the runtime registration API?